gradle/functional-test.gradle

```diff
@@ -43,4 +43,4 @@ idea.module {
 
 if (project.hasProperty('bootstrapDocker') && Boolean.parseBoolean(project.getProperty('bootstrapDocker'))) {
     apply from: "$rootDir/gradle/vagrant.gradle"
-}
\ No newline at end of file
+}
```

Comment:
this is your editorconfig that is configured to do so : "insert_final_newline = true"